### ISSUE TITLE
Expose a variable that informs host application about the currently selected section

### ISSUE THREAD

**Issue by dregatos** [nedap/segnify-ios#28]
**Description**
Describe the feature here.

As a host application 😅
I want to be able to ask 'Segnify' which section is currently selected
So I don't have to monitor it myself.

SEE: Dossier implementation for a use-case example.

**Rules/Acceptance criteria**
* R1: Segnify should expose a variable that informs about what is the currently selected section. Manually or in code.

NOTE: Selected != Visible. It might interesting to also keep track and being able to inform about the currently visible one...

**API**
APIs it depends on.

**Design**
N/A

**Dependencies**
None

**Additional context**
Add any other information or images about the feature request here.

**Is the feature ready to start development?**
- [X] It meets our [DoR](https://github.com/nedap/healthcare-mobile/wiki/DoR).

**Comment by dregatos** [comment]
Hey @vpetrykovm, 

Good job investigating this. Likely, we forgot to close the issue or it was duplicated. I'll review it tomorrow since today I have to test the release candidate of the next version of Ons Dossier.

**Comment by vpetrykov** [comment]
@dregatos could you please comment if we can close this issue based on our discussion above?

**Comment by dregatos** [comment]
Yes, I think so. Let's close it 👍🏼

**Comment by vpetrykov** [comment]
closing based on the findings described in 
- https://github.com/nedap/segnify-ios/issues/28#issuecomment-1568010186

**Comment by dregatos** [comment]
No, we can't close it and now we need it because determine the current visible section is not working in Dutch right now in Dossier :(
